SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -11,10 +11,14 @@
 Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
 > To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
 
+## Glossary
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your Unreal game files and source code. Usually called ‘Game’ e.g. `<ProjectRoot>/Game`  
+`<YourProject>` - Name of your games `.uproject` e.g. `StarterProject.uproject`  
+
 ## Setting up the project structure
-1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `Setup.bat`. This requires authorization with your SpatialOS account.
-1. Open the `StarterProject` repo and copy everything except the `\Game\` folder alongside your `<GameRoot>` folder. 
-Your folder structure should now resemble -
+1. Navigate to the root directory of the SpatialOS Unreal GDK repository you cloned (this should be `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\`) and run `Setup.bat` in a terminal window or by double-clicking the file. This requires authorization with your SpatialOS account via a web-browser.
```

Comment:
Let's just call it "GDK repository" because we're now called The SpatialOS GDK for Unreal Engine and ain't nobody got time to say that.